### PR TITLE
add poetry proposal

### PR DESCRIPTION
**Pull request by dustinblack**
## Changes introduced with this PR

Add proposal to standardize on poetry for python projects.

---
By contributing to this repository, I agree to the [contribution guidelines](https://github.com/arcalot/.github/blob/main/CONTRIBUTING.md).

### CODE REVIEW

**Review by jaredoconnell — APPROVED** [review]

**Review by portante — APPROVED** [review]

**Review by jdowni000 — APPROVED** [review]

**Review by portante — APPROVED** [review]

File: art-decisions/proposals/2022-10-26-standardize-on-poetry.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+All python projects in Arcalot should standardize on the use of
+[poetry](https://python-poetry.org/) for dependency management and packaging.
+This standard should be built into documentation and contribution guidelines,
+included in any templates, and should be checked automatically by CI.
+
+## Benefits
+
+From @mfleader:
+
+`requirements.txt` in no way defines how a python project can be built and
```
Comment (mfleader) [inline art-decisions/proposals/2022-10-26-standardize-on-poetry.md:12]: I had no idea I had contributed so much

File: art-decisions/proposals/2022-10-26-standardize-on-poetry.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+published as a package. From [PyPA](https://www.pypa.io/en/latest/) on python
+project configuration, the `pyproject.toml` is equivalent to `requirements.txt
+&& setup.py`. If we're requiring that plugins to be publishable, and `setup.py`
```
Comment (mfleader) [inline art-decisions/proposals/2022-10-26-standardize-on-poetry.md:15]: typo, change to: "If we're requiring plugins to be publishable"

**Review by mfleader — APPROVED** [review]

**Comment by mkarg75** [comment]
lgtm

**Review by mkarg75 — APPROVED** [review]

**Review by Harshith-umesh — APPROVED** [review]

**Review by sandrobonazzola — CHANGES_REQUESTED** [review]
@dustinblack can you please address @mfleader comment https://github.com/arcalot/arcalot-round-table/pull/15#discussion_r1012133378

File: art-decisions/proposals/2022-10-26-standardize-on-poetry.md (hunk 2 above)
Comment (dustinblack) [reply art-decisions/proposals/2022-10-26-standardize-on-poetry.md]: fixed

**Review by sandrobonazzola — APPROVED** [review]

**Pull request merged**